File: typescript/02-both-are-true.ts
// ¿Ambos verdaderos?
// Fácil
// 10 pts
// Booleanos

// Enunciado
// Dados dos booleanos, devuelve true solo si ambos son true (compuerta AND).

// Ejemplo
// bothTrue(true, true)   → true
// bothTrue(true, false)  → false
// bothTrue(false, false) → false
// Puedes usar console.log() para depurar. Los resultados aparecen en la Consola de salida, no en el navegador.

function bothTrue(firstValue: boolean, secondValue: boolean): boolean {
  return firstValue && secondValue;
}

// No modificar: necesario para evaluar el resultado.
export { bothTrue };

File: python/02_both_are_true.py
"""
¿Ambos verdaderos?
Fácil
10 pts
Booleanos

Enunciado
Dados dos booleanos, devuelve true solo si ambos son true (compuerta AND).

Ejemplo
both_true(True, True)   # True
both_true(True, False)  # False
both_true(False, False) # False
Puedes usar print() para depurar. Los resultados aparecen en la Consola de salida, no en el navegador.
"""


def both_true(first_value: bool, second_value: bool) -> bool:
    return first_value and second_value
